lizmap/modules/filter/controllers/service.classic.php

```diff
@@ -199,8 +188,45 @@ public function getExtent()
         // Get data
         jClasses::inc('filter~filterDatasource');
         $f = new filterDatasource($repository, $project, $layerId);
-        $rep->data = $f->getExtent($crs, $filter);
+        $streamedData = $f->getExtent($crs, $filter);
+        return $this->streamResponse($streamedData);
+    }
 
-        return $rep;
+    /**
+     * stream $streamData in a jResponseStreamed
+     * if data is iterable
+     * otherwise simple jResponseJson.
+     *
+     * @param mixed $streamedData
+     *
+     * @return jResponseJson|jResponseStreamed response
+     */
+    protected function streamResponse($streamedData)
+    {
+        if (is_iterable($streamedData)) {
+            /**
+             * @var jResponseStreamed $response
+             */
+            $response = $this->getResponse('streamed');
```

Comment:
streamed is deprecated, its new features are backported into binary, since few days. 
You can just modify : `$response = $this->getResponse('binary');`